tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
The JSON format doesn't support comments, hence this file isn't actually valid.

Reply:
removed and added arguments in `gulpfile.js`